src/main/java/mate/academy/spring/dto/request/MovieSessionRequestDto.java

```diff
@@ -1,9 +1,12 @@
 package mate.academy.spring.dto.request;
 
 import java.time.LocalDateTime;
+import javax.validation.constraints.NotNull;
 
 public class MovieSessionRequestDto {
+    @NotNull
     private Long movieId;
+    @NotNull
     private Long cinemaHallId;
```

Comment:
```suggestion
    @Min(1)
    private Long movieId;
    @Min(1)
    private Long cinemaHallId;
```
those are supposed to be ids o entities. They should be >= 1